Infrastructure/Storage/Repositories/CustomerRepository.cs

```diff
@@ -58,8 +58,8 @@ public static void AddCustomer(Customer customer)
         /// <param name="cmd"></param>
         internal static int AddCustomer(Customer customer, SQLiteCommand cmd)
         {
-            cmd.CommandText = "INSERT INTO " + TableName + " (ContragentName, Code, Entity, ContactInfoId, Description) "
-                            + "VALUES (@ContragentName, @Code, @Entity, @ContactInfoId, @Description); "
+            cmd.CommandText = "INSERT INTO " + TableName + " (ContragentName, Code, Entity, ContactInfoId, Description, IsActive) "
```

Comment:
IsEnabled or even just 'Enabled'
Conventionally, this field is just like this without 'is' part, especially in database.

Reply:
edited